src/deluge/model/global_effectable/global_effectable_for_clip.cpp

```diff
@@ -112,7 +112,7 @@ void GlobalEffectableForClip::renderOutput(ModelStackWithTimelineCounter* modelS
 	static StereoSample globalEffectableBuffer[SSI_TX_BUFFER_NUM_SAMPLES] __attribute__((aligned(CACHE_LINE_SIZE)));
 
 	bool canRenderDirectlyIntoSongBuffer =
-	    !isKit() && filterSets[0].isLPFOn() && !filterSets[1].isHPFOn() && !delayWorkingState.doDelay
+	    !isKit() && !filterSets[0].isLPFOn() && !filterSets[1].isHPFOn() && !delayWorkingState.doDelay
```

Comment:
This bug is why I suggested centralizing the check, but I'm not too attached to the idea.

Reply:
added an isOn() method